core/src/memory.rs

```diff
@@ -330,7 +330,7 @@ impl AllocationsContext {
 
         while found < pages {
             if candidate + pages > self.max_pages {
-                log::debug!(
+                log::error!(
```

Comment:
Why this should be not `debug`?  It's like user-space stuff, which doesn't show that something in platform goes wrong